data/json/professions.json

```diff
@@ -397,19 +397,23 @@
             }
         ],
         "items": {
-            "both": [
-                "tank_top",
-                "hoodie",
-                "pants_cargo",
-                "socks",
-                "sneakers",
-                "weed",
-                "tobacco",
-                "lighter",
-                "rolling_paper",
-                "switchblade",
-                "picklocks"
-            ],
+            "both": {
```

Comment:
These changes are not related to the purpose of this PR. Please try to avoid unrelated changes in future, since it increases size of PR and move the attention focus from the initial aim of PR.